features/empty.feature

```diff
@@ -8,8 +8,6 @@ Feature: Empty
     Then the exit code should be '0'
     And stdout should contain JSON matching:
     """
```

Comment:
@peter-reperger presumably the output is not valid JSON if it does not contain an empty array?